rs3_trading/database/ge_tick_database.py

```diff
@@ -3,18 +3,16 @@
 
 
 def create_price_table(con: DuckDBPyConnection):
-    con.execute('create table if not exists rs3_price_data(time datetime, item varchar, item_name varchar, price long, primary key(item, time))')
+    con.execute('create table if not exists rs3_price_data(datetime_utc datetime, item varchar, item_name varchar, price long, primary key(item, datetime_utc))')
 
 
 def create_volume_table(con: DuckDBPyConnection):
-    con.execute('create table if not exists rs3_volume_data(time datetime, item varchar, item_name varchar, volume long, primary key(item, time))')
+    con.execute('create table if not exists rs3_volume_data(datetime_utc datetime, item varchar, item_name varchar, volume long, primary key(item, datetime_utc))')
```

Comment:
Context manager? or not needed

Reply:
```
def create_database():
    with DuckDBCM(file_name=DATABASE_NAME) as con:
        create_price_table(con)
        create_volume_table(con)
```

That is the function calling the create table functions. Do I still need the context manager there?

Reply:
Nope you're good.